src/components/common/datagrid.js

```diff
@@ -90,9 +90,13 @@ const StyledCell = styled.div`
   height: 100%;
 `;
 
-const CellTemplate = ({name, type}) => {
-  const value = type === ALL_FIELD_TYPES.boolean ? String(name) : name;
-  return (<StyledCell title={value}><span>{value}</span></StyledCell>);
+const CellTemplate = ({value, type}) => {
+  const content = type === ALL_FIELD_TYPES.boolean ? String(value) : value;
+  return (
+    <StyledCell title={content}>
```

Comment:
why `title`?

Reply:
title will add the tooltip effect on mouse hover